vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-VMMServer #{ipaddress}; "
```

Comment:
Either, it's very useful to know that `Get-VMMServer "::1";` while "[::1]" works